blocks/library/table/editor.scss

```diff
@@ -1,9 +1,4 @@
 .editor-block-list__block[data-type="core/table"] {
-
-	.editor-block-toolbar__group > div {
-		display: flex;
-	}
```

Comment:
Why remove this?

Reply:
Because this `.editor-block-toolbar__group` is an editor's class defined outside of the current block. It's better to avoid it. And also, in my testing, removing this has no effect.